tests/check_framework/urls/non_unique_namespaces.py

```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import include, url
+
+common_url_patterns = ([
+    url(r'^app-ns1/', include([])),
+    url(r'^app-url/', include([])),
+], 'app-ns1')
+
+urlpatterns = [
+    url(r'^app-ns1-0/', include(common_url_patterns)),
+    url(r'^app-ns1-1/', include(common_url_patterns)),
+    url(r'^app-some-url/', include([])),
+    url(r'^some/url/$', lambda req: req, name='some_url'),
```

Comment:
Does this url affect the test in anyway (and ditto below)? If not, I think it should be removed, as test cases should be as simple as possible